crates/sui-types/src/governance.rs

```diff
@@ -28,6 +28,7 @@ pub const ADD_DELEGATION_LOCKED_COIN_FUN_NAME: &IdentStr =
 pub const WITHDRAW_DELEGATION_FUN_NAME: &IdentStr = ident_str!("request_withdraw_delegation");
 
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Good catch!

Reply:
actually we no longer expose StakedSui via RPC, we can remove the serde tag